pingpong/lti/platforms/base.py

```
@@ -0,0 +1,143 @@
+    ) -> LTIClass | Class | None:
+        """Return the existing LTIClass/Class for this (registration, course_id),
+        applying any platform-specific cross-registration lookup. Return None
+        if no class has been linked yet (caller will create a pending LTIClass).
+        """
+
+
+__all__ = [
+    "LTIPlatformHandler",
+    "parse_context_memberships_url",
+    "get_claim_object",
+]
```

Ruff (0.15.10)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)